bound/kt/src/main/kotlin/web5/sdk/dids/methods/dht/DidDht.kt

```diff
@@ -1,78 +1,84 @@
 package web5.sdk.dids.methods.dht
 
-import web5.sdk.crypto.keys.Jwk
-import web5.sdk.crypto.signers.Signer
-import web5.sdk.dids.Did
-import web5.sdk.dids.Document
+import web5.sdk.crypto.keys.KeyManager
+import web5.sdk.crypto.keys.ToInnerKeyManager
+import web5.sdk.dids.BearerDid
 import web5.sdk.dids.ResolutionResult
+import web5.sdk.rust.ServiceData
+import web5.sdk.rust.VerificationMethodData
 import web5.sdk.rust.didDhtResolve as rustCoreDidDhtResolve
-import web5.sdk.rust.DidDht as RustCoreDidDht
-import web5.sdk.rust.Signer as RustCoreSigner
+
+data class DidDhtCreateOptions(
+    val publish: Boolean? = false,
```

Comment:
so we don't publish by default? Should maybe talk to gabe, I'm 95% sure we will run into user error on this a lot. people will do default did dht create and wonder why it is not working for resolve

Reply:
> people will do default did dht create and wonder why it is not working for resolve

I would argue it is good for people to notice it is not working as they might expect and wonder why, causing them to learn that did:dht is a published did. If they do not understand that aspect of did:dht, they cannot make an informed choice of which did method to use.

Reply:
it actually is set to publish by default [as you can see here](https://github.com/TBD54566975/web5-rs/blob/21a38dba3fef3dbfc1c2596a0200342cc735208b/crates/web5/src/dids/methods/did_dht/mod.rs#L106) 

I was following [the web5-js design](https://github.com/TBD54566975/web5-js/blob/34590b2d50dbfff4c60747ce23c33fdd155a4366/packages/dids/src/methods/did-dht.ts#L616)

but perhaps here I should reinforce that experience by seeing `= true`? what do we think? Rust doesn't have the equivalent syntax so I was following Rust

Reply:
alright, that's changed in the latest commit. I think we should set it as `true` at that layer, even though the default is `true` inside the rust core code (as I linked to above), because developers often look at the function signature to determine default behavior. good catch!

Reply:
I support publish by default

Reply:
oh I just realized the way I had it before, there are code paths where the default value you be false actually, even tho I'm using the default value of true inside the rust core code. jumping across languages is hard! thanks for the spot @nitro-neal